pymrcpp/PyBoundingBox.h

```diff
@@ -0,0 +1,20 @@
+/*
+ *  \date Apr 25, 2018
+ *  \author Magnar Bjørgve <magnar.bjorgve@uit.no> \n
+ *          Hylleraas Centre for Quantum Molecular Sciences \n
+ *          UiT - The Arctic University of Norway
+ */
+
+#pragma once
+
+#include "pybind11/pybind11.h"
+#include "pybind11/numpy.h"
+
+#include "trees/BoundingBox.h"
+
+template <int D>
+class PyBoundingBox : public mrcpp::BoundingBox<D> {
+public:
+    using mrcpp::BoundingBox<D>::BoundingBox;
```

Comment:
The 'using mrcpp::' line?

Reply:
Yes.

Reply:
I use it to inherit the constructors of mrcpp::BoundingBox. Although I think I read somewhere that you no longer need to do this in c++11. But yes the compilers snaps if I don't. It is used in this example http://pybind11.readthedocs.io/en/stable/advanced/classes.html?highlight=PyAnimal . So it might be some pybind11 thing